src/components/MARCTable/MARCTableRow.js

```diff
@@ -608,4 +653,5 @@ MARCTableRow.defaultProps = {
   indicator1: '',
   indicator2: '',
   subfieldsData: [{}],
+  dispatchOnChange: () => {},
```

Comment:
I suggest using lodash noop util here.
```suggestion
  dispatchOnChange: noop,
```

Reply:
refactored